sql/src/test/java/io/crate/operation/reference/sys/node/local/SysNodesExpressionsOnHandlerTest.java

```diff
@@ -261,8 +255,8 @@ public void testNetwork() throws Exception {
         Map<String, Object> networkStats = (Map<String, Object>) collectExpression.value();
         assertThat(mapToSortedString(networkStats),
             is("probe_timestamp=0, tcp={" +
-               "connections={accepted=42, curr_established=42, dropped=42, embryonic_dropped=42, initiated=42}, " +
-               "packets={errors_received=42, received=42, retransmitted=42, rst_sent=42, sent=42}" +
+               "connections={accepted=0, curr_established=0, dropped=0, embryonic_dropped=0, initiated=0}, " +
```

Comment:
why are those 0 and not -1?

Reply:
this is because I don't want to change the existing behaviour.
There was this ZeroExtendedNodeInfo with was used when sigar was not available and this returned 0 for network stats values.

Reply:
Then maybe change the changelog entry to make it that clear?